src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -0,0 +1,275 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import azure.cli.command_modules.backup.custom_help as custom_help
+# pylint: disable=import-error
+
+from knack.log import get_logger
+
+from azure.mgmt.recoveryservicesbackup.models import AzureVMAppContainerProtectionContainer, ProtectionContainerResource
+
+from azure.cli.core.util import CLIError, sdk_no_wait
+from azure.cli.command_modules.backup._client_factory import (
+    backup_protected_items_cf, backup_protection_containers_cf, protectable_containers_cf)
+
+logger = get_logger(__name__)
+
+fabric_name = "Azure"
+default_policy_name = "DefaultPolicy"
+os_windows = 'Windows'
+os_linux = 'Linux'
+password_offset = 33
+password_length = 15
+
+# Mapping of workload type
+workload_type_map = {'MSSQL': 'SQLDataBase',
+                     'SAPHANA': 'SAPHanaDatabase'}
+
+
+def show_wl_policy(client, resource_group_name, vault_name, name):
+    return [client.get(vault_name, resource_group_name, name)]
+
+
+def list_wl_policies(client, resource_group_name, vault_name, workload_type, backup_management_type):
+    if workload_type is None:
+        raise CLIError(
```

Comment:
make a common function if this CLI error is being used multiple times in the file

Reply:
Yeah I tried to last week but error message may be changed so waiting for them to be confirmed.